Makefile

```diff
@@ -114,34 +114,28 @@ docker-buildx: ## Build and push docker image for the manager for cross-platform
 	- $(CONTAINER_TOOL) buildx rm k8s-pdb-autoscaler-builder
 	rm Dockerfile.cross
 
-.PHONY: build-installer
-build-installer: manifests generate kustomize ## Generate a consolidated YAML with CRDs and deployment.
-	mkdir -p dist
-	cd config/manager && $(KUSTOMIZE) edit set image paulgmiller/k8s-pdb-autoscaler=${IMG}
-	$(KUSTOMIZE) build config/default > dist/install.yaml
-
 ##@ Deployment
 
 ifndef ignore-not-found
   ignore-not-found = false
 endif
 
-.PHONY: install
-install: manifests kustomize ## Install CRDs into the K8s cluster specified in ~/.kube/config.
-	$(KUSTOMIZE) build config/crd | $(KUBECTL) apply -f -
+.PHONY: helm-install
+helm-install: ## Install the controller using Helm.
+	helm upgrade --install eviction-autoscaler ./helm/eviction-autoscaler \
+		--namespace kube-system --create-namespace \
+		--set image.repository=paulgmiller/k8s-pdb-autoscaler \
+		--set image.tag=${IMG#*:}
 
-.PHONY: uninstall
-uninstall: manifests kustomize ## Uninstall CRDs from the K8s cluster specified in ~/.kube/config. Call with ignore-not-found=true to ignore resource not found errors during deletion.
-	$(KUSTOMIZE) build config/crd | $(KUBECTL) delete --ignore-not-found=$(ignore-not-found) -f -
+.PHONY: helm-uninstall
+helm-uninstall: ## Uninstall the controller using Helm.
+	helm uninstall eviction-autoscaler --namespace kube-system
 
 .PHONY: deploy
-deploy: manifests kustomize ## Deploy controller to the K8s cluster specified in ~/.kube/config.
-	cd config/manager && $(KUSTOMIZE) edit set image paulgmiller/k8s-pdb-autoscaler=${IMG}
-	$(KUSTOMIZE) build config/default | $(KUBECTL) apply -f -
+deploy: helm-install ## Deploy controller to the K8s cluster specified in ~/.kube/config.
```

Comment:
this was only different than install in that it took an img. either just remove or replicate that?